app/views/posts/show.html.erb

```diff
@@ -1,24 +1,13 @@
-<div class="users-list">
- <div class="single-post"> 
- Post: <%= @post.title %> Created by <%= @post.author.name %> | comments:<% if @post.comments_counter.to_i > 0 %>
-  <%= @post.comments_counter %>
-<% else %>
-  0
-<% end %>
-, likes :<% if @post.likes_counter.to_i > 0 %>
-  <%= @post.likes_counter %>
-<% else %>
-  0
-<% end %> <br><br>
-<%= @post.text %> <br>
-
-</div>
-<div class="single-post">
-<h3>
-Comments
-</h3>
-<% @post.all_comments.each do |comment|%>
-<%= comment.author.name %>: <%= comment.text %> <br>
-
-<% end %></div>
- </div>
+<article class="post">
+  <h3><%= @post.title %>  by <%= @post.author.name %></h3>
+  <div class="post-stats">
+    <span>Comments: <%= @post.comments_counter %> &nbsp;,&nbsp;</span>
+    <span>Likes: <%= @post.likes_counter %></span>
+  </div>
+  <p><%= @post.text %></p>
+</article>
+<ul class="comments">
+  <% @post.recent_comments.each do |comment| %>
```

Comment:
- [ ] Great job woking on it but please make sure to have the same wireframe as I asked previously  it would be great if you make you layout the same as the given wireframe no styles at this moment but the wireframe it should be the same so kindly work on it to have it similar to the given one 🔥❤🚀


- [ ] I think there is an error when entering a new comment  so could please make sure to fix the problem add the required view to display some comments it is like this for me ⬇⬇

![image](https://user-images.githubusercontent.com/58553711/231932465-deb7c8a0-9d8a-4230-a7e3-c76efb9758b5.png)


- It should looks like
![image](https://user-images.githubusercontent.com/58553711/231932518-c9ff214f-14ce-43eb-9414-7cde58886d62.png)